docs/docker.md

```diff
@@ -1,11 +1,11 @@
 # Dockerfiles
 
-Our test runners are deployed as docker images.
+Our test runners are deployed as Docker images.
 
-- Your dockerfile should live at the root directory of your repository and it should be called `Dockerfile`
-- Your dockerfile should be the minimal needed for your test runners and create the minimal image needed for running the tests.
-  It should use alpine linux if possible.
-- The working directory should be `/opt/test-runner`
+- Your dockerfile should live at the root directory of your repository and it should be called `Dockerfile`.
+- Your dockerfile should create the minimal image needed for your test runner to run the tests.
+  It should use alpine linux if possible. Applying the official [Dockerfile best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) can help to create a minimal image.
```

Comment:
Do we actually care about alpine linux? Or is it purely that it's small. Should we flip this and say "Applying the official..." first and then "The minimal-sized image is often based on alpine linux"

Reply:
I think that makes sense. Let me update.